react/src/containers/SiteShowContainer.js

```diff
@@ -73,6 +164,27 @@ class SiteShowContainer extends Component {
             experience = {this.state.site.experience}
           />
         </div>
+
+      <div className="column row">
+        <NewReviewForm
+          className = {className}
+          handleFormButtonClick = {this.handleFormButtonClick}
+          userValue = {this.state.user_id}
+          ratingValue = {this.state.overall_rating}
+          designValue = {this.state.design_body}
+          usabilityValue = {this.state.usability_body}
+          conceptValue = {this.state.concept_body}
+
+          ratingChange = {this.handleRatingChange}
+          userChange = {this.handleUserIdChange}
+          designChange = {this.handleDesignChange}
+          usabilityChange = {this.handleUsabilityChange}
+          conceptChange = {this.handleConceptChange}
+
+          handleSubmit = {this.handleSubmit}
+        />
+      </div>
+
```

Comment:
this seems a little extreme. you would probably be much better off moving much of the state that these methods are changing down to the NewReviewForm container, and making that a stateful component.